uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/calendar-ctrl.js

```diff
@@ -334,14 +341,16 @@ define([
      * then hide each loader on the interval of an offset value
      */
     this._showMonthLoader = function () {
-      var monthLoadDelay = 500,
-        offset = 0;
+      var monthLoadDelay,
```

Comment:
same var rule applies here also. declaration and assignment should be in same line, please check all instances